File: Uri-Iniciante/1035.cpp
#include<stdio.h>
/*	
*	-----------------------------------
*	|		Pedro Daniel Jardim       |
*	| 			  UFV				  | 
*	|		  26/02/2020			  |
*   ----------------------------------
*
*/



int main() 
{
    int A,B,C,D;
	scanf("%d %d %d %d",&A,&B,&C,&D);
	if( (B>C)&&(D>A)&&(C+D>A+B)&&(C>=0)&&(D>=0)&&(A%2==0)) {
		printf("Valores aceitos");
	}else{
		printf("Valores nao aceitos");
	}
   return 0;
}

File: Beginner/1035.py

A = int(input())
B = int(input())
C = int(input())
D = int(input())

if B > C and D >A and C+D > A+B and C >= 0 and D >= 0 and A % 2 == 0:
    print("Valores Aceitos")
else:
    print("Valores não aceitos")